java/src/jmri/jmrit/logix/Engineer.java

```diff
@@ -180,6 +180,7 @@ public void run() {
                 if (command.equals("SPEED")) {
                     float speed = Float.parseFloat(ts.getValue());
                     _lock.lock();
+                    if (log.isTraceEnabled()) log.trace("SPEED CMD: speed= {} type= \"{}\"", speed, _speedType);
```

Comment:
This is just a passing comment, no need to change anything, but the "if (log.isTraceEnabled())" tests before the log.trace statements generally aren't needed (they used to be, but we changed logging frameworks and now they mostly aren't).